.github/workflows/test_full.yml

```diff
@@ -74,5 +74,5 @@ jobs:
       - name: run pytest
         run: |
           source <(cat .env_file | sed 's/=/=/' | sed 's/^/export /')
-          pytest --tb=long -vv --cache-clear --no-cov --allow-skip=all tests/
+          pytest -m "not slow" --tb=long -vv --cache-clear --no-cov --allow-skip=all tests/
```

Comment:
I would expect `test_full` job to run **all** tests, not **all but for slow ones**.

Reply:
Removed the limitation.